src/js/components/organisms/content-cover/content-cover.unit.spec.ts

```diff
@@ -0,0 +1,34 @@
+/** @format */
```

Comment:
This whole file doesn't add anything and should be removed from the hygen generator.

Reply:
Basic unit test, not going to remove that.